cpp/src/memory/memory_manager.cu

```diff
@@ -74,4 +68,53 @@ void get_pointer_attributes(PointerAttributes& attr, const void* ptr)
     }
 }
 
+bool move_raster_from_host(void** target, size_t size, cucim::io::Device& dst_device)
```

Comment:
Can device be `const`?

```suggestion
bool move_raster_from_host(void** target, size_t size, const cucim::io::Device& dst_device)
```

Reply:
Yes, it should be `const`. Will update it.